src/server/pps/server/server.go

```diff
@@ -78,6 +78,7 @@ func NewAPIServer(env Env) (ppsiface.APIServer, error) {
 	apiServer := (srv).(*apiServer)
 	apiServer.validateKube(apiServer.env.BackgroundContext)
 	go apiServer.master()
+	go apiServer.worker()
```

Comment:
I think I would prefer to leave it as is for now because I don't think we will have other types of workers and I think it would make more sense to split up the PPS master similarly to what we are doing with the PFS master.